### PR TITLE
Fix _cache_founf_inf

### PR DESCRIPTION
**Pull request by co63oc**
### PR types

Others 
### PR changes

Others 
### Description

Fix _cache_founf_inf

Paddle也需要修改 https://github.com/PaddlePaddle/Paddle/pull/61954

### CODE REVIEW

File: examples/RLHF/ppo_trainer.py [hunk 1]
```diff
@@ -415,7 +415,7 @@ def full_training_step(self: Trainer, inputs: Dict[str, paddle.Tensor], **kwargs
             self.scaler.step(self.optimizer)
             self.scaler.update()
             scale_after = self.scaler._scale
-            optimizer_was_run = not self.scaler._cache_founf_inf
+            optimizer_was_run = not self.scaler._cache_found_inf
```
Comment (wawltor) [inline examples/RLHF/ppo_trainer.py]: Please check if there's a way to ensure compatibility between interfaces. We've encountered an issue where using PaddleNLP's development version with PaddlePaddle v2.6.0 results in incompatibility.

Comment (co63oc) [reply examples/RLHF/ppo_trainer.py]: image
Changed to check _cache_founf_inf

**Comment by co63oc** [comment]
@wawltor It has been modified. codecov suggested that there was a coverage problem. Do I need to add the test, Where is it to add

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**